roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -56,12 +56,13 @@ kiali_defaults:
   custom_dashboards: []
 
   deployment:
-    accessible_namespaces: ["^((?!(istio-operator|kube-.*|openshift.*|ibm.*|kiali-operator)).)*$"]
+    accessible_namespaces: ["**"]
```

Comment:
this is the default yaml - unless there is a specific reason why, we set everything in there to avoid making tests in the ansible scripts everywhere with "if this is unset then do this, otherwise, do that".

This is internal implementation details - users shouldn't care. This is the default you are asking for (i.e. if AN is unset, this is where the operator assumes `**`).

Reply:
OK, I suppose if it's our default we have to keep it here, and we need it handled for backward compatibility.  But outside of that I don't think we need to document `**`, just the behavior of when it is unset.